src/pages/projectView/projectMainBody/components/ProjectRewardPanel.tsx

```diff
@@ -25,11 +25,22 @@ export const ProjectRewardPanel = ({ reward }: Props) => {
     fundForm: { updateReward },
   } = useProjectContext()
   const navigate = useNavigate()
+  const rewardStockRemaining = reward.maxClaimable ? reward.maxClaimable - reward.sold : -1;
 
   if(!project || !isActive) {
     return <></>;
   }
 
+  const renderRewardAvailability = () => {
+    if(rewardStockRemaining > 0) {
+      return <><span style={{color: '#DF3634'}}>{rewardStockRemaining + ` ${t('remaining')}`}</span> <span style={{fontSize: "10px", position: "relative", top: "-2px"}}>&#8226;</span> </>;
+    } else if (rewardStockRemaining === 0) {
+      return <><span style={{color: 'neutral.600', fontWeight: 700}}>{t('Sold Out')}</span> <span style={{fontSize: "10px", position: "relative", top: "-2px"}}>&#8226;</span> </>;
+    } else {
+      return '';
+    }
+  }
+
```

Comment:
This whole section looks repeated to the first file in this PR. The same comments as there.